.github/workflows/Packages - EdFi.Ods.Extensions.Homograph.yml

```diff
@@ -0,0 +1,93 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: Packages - EdFi.Ods.Extensions.Homograph
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main      
+  workflow_dispatch:
+
+env:
+  INFORMATIONAL_VERSION: "6.1"
+  BUILD_INCREMENTER: "20"
+  CONFIGURATION: "Release"
+  CURRENT_BRANCH:  ${{ GITHUB.REF_NAME }}
```

Comment:
Is this `CURRENT_BRANCH` env variable being used anywhere? I believe it was used in the previous workflows with the way we checked out the other repos before

Reply:
removed it